docs/guides/primitive-input-output.ipynb

```diff
@@ -356,6 +356,28 @@
     "print(f'The expectation values measured from this PUB are: \\n{result[0].data.evs}')"
```

Comment:
<p>In that final sentence, </p><blockquote>When ZNE is enabled, <code>pub_result.data.evs_noise_factors</code> and <code>pub_result.data.stds_noise_factors</code> is also populated </blockquote><p>it should say "are also populated".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
YES it should! good catch